console/src/pages/users/ViewUserPage.tsx

```diff
@@ -107,9 +131,12 @@ export const ViewUserPage = () => {
       </PageDescription>
 
       <Card className="my-8">
-        <CardHeader>
-          <CardTitle>General settings</CardTitle>
-          <CardDescription>Basic settings for this user.</CardDescription>
+        <CardHeader className="flex flex-row items-center justify-between space-y-4">
+          <div>
+            <CardTitle>General settings</CardTitle>
+            <CardDescription>Basic settings for this User.</CardDescription>
+          </div>
+          <EditUserSettingsButton />
```

Comment:
Just for consistency and ease of later grepping to fix this everywhere, can you please follow this convention?

https://github.com/tesseral-labs/tesseral/blob/698607eabaaf6e1cb2c2f1386e2aefcbcbca07eb/console/src/pages/api-keys/ListAPIKeysPage.tsx#L97-L108